ssc/cardano-sl-ssc.cabal

```diff
@@ -108,6 +103,7 @@ library
                      , universum
                      , unordered-containers
 
+  hs-source-dirs:      src
```

Comment:
Yeah, this is something that I specified in the `docs/code-organisation-guidelines.md` that was added to the repo back in APril. Most packages conformed to this but a few did not. And since we have that document, the external reviewers are now pinging us on it.

When ever we start work on a package we should check this and make the required changes as @Jimbo4350 has done here.